ipaserver/install/server/upgrade.py

```diff
@@ -1389,7 +1389,11 @@ def fix_trust_flags():
     if cert:
         db.trust_root_cert(nickname, 'CT,C,C')
 
-    sysupgrade.set_upgrade_state('http', 'fix_trust_flags', True)
+    sc_nickname = installutils.get_directive(paths.HTTPD_NSS_CONF,
+                                             "NSSNickname")
```

Comment:
I'm not sure whether there can be a situation when NSSNickname is not specified. Is it possible or not?